core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/BoundedRangeFileInputStream.java

```diff
@@ -95,18 +92,7 @@ public int read(final byte[] b, final int off, int len) throws IOException {
         throw new IOException("Stream closed");
       }
       in.seek(pos);
-      try {
-        ret = AccessController.doPrivileged(new PrivilegedExceptionAction<Integer>() {
```

Comment:
This is the one change which I'm not quite sure about.

It having been there since the dawn of time and without comment, I'm really not sure what it was intended to do. This predates all of the Kerberos work -- the only thing I can think of would have been the old SecurityManager constraints (but, if memory serves, those were more around SKVI's, not reading data from HDFS).

@billierinaldi, @keith-turner, @ericnewton, @scubafuchs: any guesses? :)